source/Files.cpp

```diff
@@ -561,6 +554,21 @@ void Files::Write(FILE *file, const string &data)
 
 
 
+void Files::CreateFolder(const std::string &path)
+{
+	if(Files::Exists(path))
```

Comment:
`Files::Init` doesn't seem to include the `Files::` prefix on other methods in `Files` that it calls, like `Exists` and `CreateFolder`. Any reason why this method is different? Should it always include the prefix?